### PR TITLE
Tests & Enum

### PR DESCRIPTION
**Pull request by Sav4ykVladzislau**
https://schneidervs.atlassian.net/browse/TGBOT-2

### CODE REVIEW

File: tg-bot/src/main/java/com/tutrit/tgbot/example/service/TelegramBot.java [hunk 1]
```diff
@@ -1,43 +1,78 @@
...
 import org.telegram.telegrambots.meta.api.objects.Update;
 import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
 
-@Service
-public class TelegramBot extends TelegramLongPollingBot {
+import java.io.File;
+import java.io.IOException;
+
+@Component
+@PropertySource("application.properties")
```
Comment (max-java) [inline tg-bot/src/main/java/com/tutrit/tgbot/example/service/TelegramBot.java:17]: That is not necessary, cose that is a default value

File: tg-bot/src/test/java/com/tutrit/tgbot/example/config/Mapper.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.tutrit.tgbot.example.config;
```
Comment (max-java) [inline tg-bot/src/test/java/com/tutrit/tgbot/example/config/Mapper.java:1]: class could be deleted, the one from `src` should be used

File: tg-bot/src/test/java/com/tutrit/tgbot/example/service/StatusServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+class StatusServiceTest {
+
+    @Autowired
+    TelegramBot telegramBot;
+    @Autowired
+    MessageService messageService;
+
+    @Test
+   void choice() {
+        StatusService.choice(OrderStatus.REQUEST);
```
Comment (max-java) [inline tg-bot/src/test/java/com/tutrit/tgbot/example/service/StatusServiceTest.java:16]: and?

File: tg-bot/src/test/java/com/tutrit/tgbot/example/service/TelegramBotTest.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+   telegramBot.onUpdateReceived(update);
+    }
+//    @Test
+    void sendMassage() throws TelegramApiException {
+        SendMessage sendMessage = new SendMessage();
+        sendMessage.setChatId(1035881400L);
+        sendMessage.setText("Hello bot");
+        telegramBot.execute(sendMessage);
+    }
+}
```
Comment (max-java) [inline tg-bot/src/test/java/com/tutrit/tgbot/example/service/TelegramBotTest.java:32]: add empty row

**Review by max-java — CHANGES_REQUESTED** [review]
ptal

**Comment by schneidervs** [comment]
передалай, плиз

**Comment by max-java** [comment]
outdated

**Pull request closed without merging**